### ISSUE TITLE
Turing architecture of GPU: Apollo Master and Apollo 6.0

### ISSUE THREAD

**Issue by Rachid-CETRAN** [ApolloAuto/apollo#12294]
### System information

- **OS Platform and Distribution (e.g., Linux Ubuntu 18.04)**:  Linux Ubuntu 18.04
- **Apollo installed from (source or binary)**: source with GPU enabled
- **Apollo version (3.5, 5.0, 5.5, 6.0)**: 5.5 or 6.0
- **Output of `apollo.sh config` if on `master` branch**:

### Steps to reproduce the issue:
- We would like to run Apollo 5.5 or next version 6.0 for simulation purposes. For this, we have planned to buy new workstation with good Nvidia GPU card. 
- We understood that:
     1- In the upcoming version of Apollo (Apollo 6.0), the Turing architecture of GPU card will be supported.
     2- The Master branch of Apollo support already Turing architecture of GPU but:
                                    - the Apollo master is not yet stable 
                                    - the perception is only based on LiDAR perception module
                                    - the camera perception for traffic light perception is not working 

Could you please confirm whether the above findings are correct or not? 

Could you please let us know when Apollo 6.0 is planned to be released? and what will be the recommended NVIDIA RTX to use to run Apollo 6.0?

**Comment by storypku** [comment]
Let me try to answer your question of 1 and 2(1):

1. Yes, Apollo 6.0 will come with CUDA-10.2+cuDNN-7.6+TensorRT-7.0, supporting Compute Capabilities of "5.0 6.0 6.1 7.0 7.2 7.5", covering GPU archs for Maxwell, Pascal, Volta and Turing. 
2(1): The team is working hard to improve stability and performance of the master branch to make it ready for Apollo 6.0 release. As you know, 6.0 was a breaking upgrade from previous versions.

Hope my (partial) answer helps.

**Comment by storypku** [comment]
@jeroldchen @panfengsu @xiaoMrzhang Please update the progress of the Perception module. Thanks!

**Comment by Thinsheep** [comment]
Hi,
To running Apollo 6.0
What does it mean by " NVIDIA Turing GPU is strongly recommended"? Will RTX 3080/3090 are also good and do a better job than RTX 2080/2080 Ti?

**Comment by Thinsheep** [comment]
@storypku 
you did not mention Ampere so I guess 3080/3090 are not supported to run Appolo 6.0.

**Comment by storypku** [comment]
> @storypku
> you did not mention Ampere so I guess 3080/3090 do not support to run Appolo 6.0.

Yep. Ampere support was only available in recent master. At the time of the release of Apollo 6.0, CUDA 11.x is not available then.

**Comment by marcusvinicius178** [comment]
Hi @storypku @Thinsheep  I need to run Apollo 6.0 and SVL simulator on the same machine. I rent good machines on paperspace.com ( P6000x2, RTX5000x2, V100) however they were not good enough to run the LATTICE planner in complex scenarios due the low Clock frequency, GPU power ( It displays delay error message in Inputs for the planning module and is disabled) ...they runned just the PUBLIC_ROAD planner without fails. However both planners run on simple scenario (Single_Lane_Road). 
As I wish to run simulation in complex scenarios, WHICH MACHINE (DESKTOP) do you recommend? 
Also would you know which website offers this powerful  cloud GPU desktop to rent?
